packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -428,7 +428,13 @@ const getParcelsFromAPI = async () => {
   try {
     // Fund Gnosis Safe wallet with Mana tokens and ethers
     await etherFaucet(provider, minterWallet, GNOSIS_SAFE_ADDRESS, TEN);
+    await expectExactEtherBalances(provider, [GNOSIS_SAFE_ADDRESS], [TEN]);
     await erc20Faucet(manaContract, minterWallet, GNOSIS_SAFE_ADDRESS, BILLION);
+    await expectExactTokenBalances(
```

Comment:
Are these the only tests where we were using these faucets?

Reply:
These are the ones without balances expectation after the faucet.

Reply:
Ah ok cool. Resolving this.